src/PrestaShopBundle/Bridge/Controller/ControllerBridgeInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShopBundle\Bridge\Controller;
+
+/**
+ * Define a Controller interface which can be shared with legacy
+ */
+interface ControllerBridgeInterface
+{
+}
```

Comment:
What is the purpose of this empty interface ? What is the contract here ?

Public property controllerConfiguration could be easily set as private, and this interface could offer a `getControllerConfiguration` method instead.
The existence of this property is tested everywhere in this PR, and it would become useless if the interface was properly used.

Reply:
@jolelievre 
I got confused with a few property checks in the configuration itself. My assertion was indeed very exagerated :)

IMHO the controller configuration is too sensitive to leave it open to any wild update. A setter method sounds much safer indeed. There is no reason to feel stingy about OOP safety, even for temporary stuff :)